HandheldCompanion/Controls/GameView.xaml

```diff
@@ -0,0 +1,293 @@
+                        TextWrapping="WrapWithOverflow"
+                        HorizontalAlignment="Stretch"
+                        TextAlignment="Center"
+                        Text="{Binding SelectedGame.MetaData.FullImageUrl}" />
+                        
+                        <Button                          
+                            Grid.Column="0"
+                            Grid.Row="0"
+                            Name="B_LaunchGame"
+                            HorizontalAlignment="Stretch"
+                            Command="{Binding RunGameCommand}"
+                            CommandParameter="{Binding SelectedGame}"
+                            ToolTip="Start Game"
+                            Content="Play"
+                            Style="{DynamicResource AccentButtonStyle}">
+                        </Button>
+                        <Button                          
+                            Grid.Column="0"
+                            Grid.Row="3"
+                            Name="B_IsRunning"
+                            HorizontalAlignment="Stretch"
+                            Command="{Binding RefreshGameIsRunningCommand}"
+                            ToolTip="Check Is Running"
+                            Content="Is Running?"
+                            Style="{DynamicResource AccentButtonStyle}">
+                        </Button>
+
+                        <Image
+                            Grid.Row="4"
+                            Grid.Column="0"
+                            Width="24"
+                            Height="24"
+                            HorizontalAlignment="Left"
+                            Source="{Binding IsRunningLogo}" />
+                    </Grid>
+                </Grid>
+
+            </ui:SimpleStackPanel>
+            <!--<TextBlock
+                Grid.Column="0"
+                Grid.Row="1"
+                Padding="10,10,10,10"
+                VerticalAlignment="Stretch"
+                TextWrapping="WrapWithOverflow"
+                HorizontalAlignment="Stretch"
+                TextAlignment="Left"
+                Text="All Games" />-->
+        </ui:SimpleStackPanel>
+        <ListBox 
+            Grid.Row="0"
+            d:Width="Auto"
+            d:Height="Auto"
+            Padding="0,10,0,20"
+            BorderThickness="0,0,0,0"
+            Foreground="{StaticResource PrimaryForegroundColor}"
+            ItemsSource="{Binding Games}" 
+            SelectedItem="{Binding SelectedGame,Mode=TwoWay}">
+            <ListBox.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <WrapPanel />
+                </ItemsPanelTemplate>
+            </ListBox.ItemsPanel>
+            <ListBox.ItemContainerStyle>
+                <Style TargetType="ListBoxItem">
+                    <Setter Property="Template">
+                        <Setter.Value>
+                            <ControlTemplate TargetType="ListBoxItem">
```

Comment:
Why is ItemTemplate not used here? Not a fan of doing this through style setters.

Reply:
As with most of the Page related things its just me figuring out WPF and using examples i find. Thats why i would love to see the refactor and how it should be done